content/docs/command-reference/import.md

```diff
@@ -8,9 +8,10 @@ Download a file or directory tracked by another DVC or Git repository into the
 ## Synopsis
 
 ```usage
-usage: dvc import [-h] [-q | -v] [-j <number>]
-                  [-o <path>] [--file <filename>]
-                  [--rev <commit>] [--no-exec | --no-download] [--desc <text>]
+usage: dvc import [-h] [-q | -v] [--file <filename>]
```

Comment:
if we change order, should we also update in DVC itself?

Reply:
Yes, that would be ideal cc @dberenbaum

Reply:
@jorgeorpinel Is this minor cleanup or is there a larger goal? If it's minor cleanup, would you mind submitting the accompanying core change PR in these cases?

Reply:
Def. minor. Yes, sorry I know I should just didn't have time but also didn't want to leave it as it is now and it's quick to fix on the docs side. Unfortunately we didn't catch this during which ever PR added flags that made the usage block overflow. I'll add it to the best/std practices (maybe we can make a check list) for writing/updating cmd refs.

Reply:
Thanks @jorgeorpinel! Do you plan to open the core PR (no rush, just clarifying)? I think it makes sense to set the precedent that whoever contributes a fix owns propagating the changes to both core and docs repos.

Reply:
I'll take care of it 🙂

Reply:
Thanks @jorgeorpinel!

Reply:
p.s. I just noticed that we did catch this but post-merge. See https://github.com/iterative/dvc.org/pull/3773#discussion_r964450724. There's a follow-up to that too: https://github.com/iterative/dvc.org/pull/3930 -- reviewing now.